src/Abp.Zero.Common/Organizations/OrganizationUnitManager.cs

```diff
@@ -14,10 +15,12 @@ namespace Abp.Organizations
     /// </summary>
     public class OrganizationUnitManager : DomainService
     {
+        private readonly IAsyncQueryableExecuter _asyncQueryableExecuter;
```

Comment:
@PMExtra please use property injection here over constructor injection

Reply:
It's ok but I want to know why? What's the difference?